tareekh.go

```diff
@@ -104,3 +104,15 @@ func DifferenceInSeconds(then, now time.Time) int {
 func Format(t time.Time) string {
 	return t.Format("2006/01/02 15:04:05")
 }
+
+// BeginningOfMonthForInput ...
+func BeginningOfMonthForInput(t time.Time) time.Time {
```

Comment:
`ForInput` in the function name seems a little off.. can you change it to  `ExtractBeginningOfMonth()` so it does what it says? similarly for the other function.

Reply:
fixed